src/reducers.test.js

```diff
@@ -3317,12 +3319,37 @@ describe('minePlot', () => {
   beforeAll(() => {
     gameState = {
       field: [[null, 'crop']],
+      inventory: [],
     }
+
+    jest.spyOn(ResourceFactory, 'instance')
+
+    ResourceFactory.instance.mockReturnValue({
+      generateResources: () => [goldOre],
+    })
+
     gameState = fn.minePlot(gameState, 0, 0)
   })
 
   test('updates the plot to be shoveled if the plot is empty', () => {
-    expect(gameState.field[0][0].wasShoveledToday).toEqual(true)
+    expect(gameState.field[0][0].wasShoveled).toEqual(true)
+  })
+
+  test('sets the oreId on the plot if ore was spawned', () => {
+    expect(gameState.field[0][0].oreId).toEqual(goldOre.id)
+  })
+
+  test('adds the spawned ore to the inventory', () => {
+    let itemIsInInventory = false
+
+    for (let item of gameState.inventory) {
+      if (item.id === goldOre.id) {
+        itemIsInInventory = true
+        return
```

Comment:
It looks like this should be `break` so that the `expect` below is always hit.

Reply:
oh yeah good catch